figures/horizontal.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
```

Comment:
My point wasn't to remove the import statement, but the whole file since it just a copy of `line.py` and the name `horizontal` is misleading.

Reply:
Ah lol. My bad